core-web/libs/edit-content/src/lib/components/dot-edit-content-field/dot-edit-content-field.component.ts

```diff
@@ -2,15 +2,20 @@ import { CommonModule } from '@angular/common';
 import { ChangeDetectionStrategy, Component, Input, inject } from '@angular/core';
 import { ControlContainer, ReactiveFormsModule } from '@angular/forms';
 
-import { InputTextModule } from 'primeng/inputtext';
-
 import { DotCMSContentTypeField } from '@dotcms/dotcms-models';
 import { DotFieldRequiredDirective } from '@dotcms/ui';
 
+import { DotEditContentFieldsModule } from '../../fields/dot-edit-content-fields.module';
+
 @Component({
     selector: 'dot-edit-content-field',
     standalone: true,
-    imports: [CommonModule, ReactiveFormsModule, InputTextModule, DotFieldRequiredDirective],
+    imports: [
+        CommonModule,
```

Comment:
Do you need all the commons?

Reply:
No, I'll import the ones that we need here